docs/ko/docs/tutorial/query-params-str-validations.md

```diff
@@ -0,0 +1,304 @@
+# 쿼리 매개변수와 문자열 검증
+
+**FastAPI**를 사용하면 매개변수에 대한 추가 정보 및 유효성 검사를 선언할 수 있습니다.
+
+이 애플리케이션을 예로 들어보겠습니다:
```

Comment:
```suggestion
이 응용 프로그램을 예로 들어보겠습니다:
```

If you check [파이썬 한국어 번역](https://www.flowdas.com/pages/python-docs-ko.html) and search word, `application`, it is translated as `응용 프로그램` and because we follows the rules of that guide (check of it on #2017 ).